### DISCUSSION TITLE
2022-04-15 Data Platform Team Updates

### DISCUSSION THREAD

**Discussion by khorolets** [near/community#247] [Archived]
👋 Wow, that was a very quick week. Hello everyone! Data Platform Team is here for the weekly updates. Unfortunately didn't succeed in publishing the Indexer Docs this week, because we're dealing with dedicated domain stuff. I really hope we'll do it next week. 

Our team has created a draft OKRs for the 2022 Q2 and soon enough it will be finalized, so we've defined our focus for the next quarter 🎉

Our main focus remains the same:

- @telezhnaya keeps working on the new alternative version of Indexer for Explorer with a working name [NEAR Lake Flow into SQL Base](https://github.com/telezhnaya/near-lake-flows-into-sql-base)
- @khorolets and @frol are currently working on publishing the Indexer Docs. Meanwhile, the semi-ready docs are [published with a personal GitHub account](https://khorolets.github.io/near-lake-dock/docs/intro), but it's going to be moved from there very soon
- Also, we're filling Indexer Docs with actually docs and articles (@khorolets)
- For those who wonder, NEAR Lake Framework JavaScript is in development. Some tech details if you're curious, we're using TypeScript and now we need to provide a `StreamerMessage` type with annotations, it is huge and we want to do everything properly, so the developers wouldn't struggle using those. Thanks to everybody who waits patiently, we're working on it (@khorolets)

Though not everything we wanted has been achieved this week, we keep working, we keep planning, we keep rocking 🤘 That's all for now. Don't forget to have a rest. And have a great weekend! See you all next week!

**Comment by idea404** [comment]
YT video something for next week? :)

**Reply by khorolets** [reply]
Yeah. I guess we'll publish it on Monday even without docs

**Reply by khorolets** [reply]
@idea404 here you go https://www.youtube.com/watch?v=VSBJ-A69Km4
